apps/web/components/dashboard/tags/AllTagsView.tsx

```diff
@@ -104,22 +187,73 @@ export default function AllTagsView({
   const tagsToPill = (tags: typeof allTags) => {
     let tagPill;
     if (tags.length) {
-      tagPill = tags.map((t) => (
-        <TagPill key={t.id} id={t.id} name={t.name} count={t.count} />
-      ));
+      tagPill = (
+        <div className="flex flex-wrap gap-3">
+          {tags.map((t) => (
+            <Draggable
+              key={t.id}
+              axis="both"
+              onStart={handleDragStart}
+              onDrag={handleDrag}
+              onStop={handleDragEnd}
+              disabled={!draggingEnabled}
+              defaultClassNameDragging={
+                "position-relative z-10 pointer-events-none"
+              }
+              position={
+                !currentState.dragId
+                  ? { x: currentState.x ?? 0, y: currentState.y ?? 0 }
+                  : undefined
+              }
+            >
+              <div
+                className="group relative flex cursor-grab"
+                data-id={t.id}
+                onDragOver={handleDrag}
+              >
```

Comment:
How about we pass `onDragOver` to `TagPill` and remove that div?

Reply:
turns out the onDragOver is not actually needed, so I removed it.
If I remove the div though, it is suddenly not possible to drag it anymore. I am not sure why though...